ancar/portfolio/src/main/java/com/google/sps/servlets/MapServlet.java

```diff
@@ -0,0 +1,54 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Returns map data as a JSON array, e.g. [{"lat": 38.4404675, "lng": -122.7144313}] */
+@WebServlet("/map-data")
+public class MapServlet extends HttpServlet {
+
+    private class elementMap {
+        double lat, lng;
+        String name, description;
+
+        public elementMap(double lat, double lng, String name, String description) {
+            this.lat = lat;
+            this.lng = lng;
+            this.name = name;
+            this.description = description;
+        }
+    }
+
+    private ArrayList<elementMap> markers;
+
+    @Override
+    public void init() {
+        markers = new ArrayList<elementMap>();
+
+        Scanner scanner = new Scanner(getServletContext().getResourceAsStream("/WEB-INF/cities.csv"));
+        while (scanner.hasNextLine()) {
+            String line = scanner.nextLine();
+            String[] cells = line.split(",");
+
+            double lat = Double.parseDouble(cells[0]);
+            double lng = Double.parseDouble(cells[1]);
+
+            markers.add(new elementMap(lat, lng, cells[2], cells[3]));
+        }
+        scanner.close();
```

Comment:
Please use [try with resources](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html), so it will automatically call 
"close" method of the scanner when the job is done or when
an exception is thrown.

```java
try (Scanner scanner = new Scanner(getResourceAsStream("..."))) { 
}
```